### DISCUSSION TITLE
江湖救急（但是不是ci的问题）

### DISCUSSION THREAD

**Discussion by MKStoler4096** [ClassIsland/ClassIsland#491] [Q&A]
帮我下载一下这个~~极为傻逼的~~CSDN资源，然后帮忙发我
感谢感谢，要用来整活（）
如你所见，[**这是个~~无耻的~~链接**](https://download.csdn.net/download/julielele/87898957)，竟然要登录账号。

**Comment by MKStoler4096** [comment]
要不是我和WRC一样蹲大牢就自己下载了（）

**Comment by wjj-8283** [comment]
也许可以自己编译？python可以加载c的代码，而uiaccess是开源的

**Comment by MKStoler4096** [comment]
不是，发错地方了，才注意到发到这里了（
（我说怎么写的不对劲）
